testdata/dnn/darknet/crop.cfg

```diff
@@ -0,0 +1,23 @@
+[net]
+# Training
+# batch=128
+# subdivisions=4
+
+# Testing
+batch=1
+subdivisions=1
+
+height=256
+width=256
+channels=3
```

Comment:
Please choose smaller input resolution I think the following is enough for the test:
```
height=5
width=6
channels=3
```
```
crop_height=3
crop_width=2
```